File: Number_Guessing_Game.py
from time import sleep
import random
N= random.randint(0,100)
X=-1
i=1
while X!=N:
    X=int(input("Enter a number \n"))
    if X==N:
        print("yes you are right ")
    else:
        if X>N:
            print("Lower!")
        else:
            print("Higher")
        i=i+1
sleep(0.5)
if i==1:
    print("you did it from the frist time")
else:
    print(" you did it after",i,"attempts")

File: Random_Number_Generator/Number_Guessing_Game.js
let button=document.getElementById("button");
let RandomNumber=Number(Math.floor(Math.random()*101));
console.log(RandomNumber);
button.onclick=function(){
    let GuessedNumberr=Number(document.getElementById("input").value);
        {if(GuessedNumber==RandomNumber)
            {document.getElementById("label").innerText="You Guessed Right!!!!";
            RandomNumber=Number(Math.floor(Math.random()*100)+1);
            console.log(RandomNumber);
            document.getElementById("input").value=""
        }
        else
        {
           document.getElementById("input").value=""
           if(GuessedNumber>RandomNumber)
           {
            document.getElementById("label").innerText="Try Again!!!! its Lower";
           }else{
            document.getElementById("label").innerText="Try Again!!!! its Higher";
           }
        }
    }
}
